CHANGELOG.md

```diff
@@ -20,5 +20,9 @@
 - \(GH-16\) Do not warn on "\s" string literal [\#20](https://github.com/puppetlabs/puppet-lint/pull/20) ([sanfrancrisko](https://github.com/sanfrancrisko))
 - \(maint\) Prepare repo and gem for next release [\#15](https://github.com/puppetlabs/puppet-lint/pull/15) ([da-ar](https://github.com/da-ar))
 
+## [2.5.1](https://github.com/puppetlabs/puppet-lint/tree/2.5.1) (2021-09-13) [YANKED]
+
+**This release was yanked as there were unexpected artefacts published in the build**
```

Comment:
odd. Why are the closed issues/fixed bugs listed in the 2.5.2 release and not 2.5.1?

Reply:
Given 2.5.1 has been yanked, I'm going to be deleting the 2.5.1 tag and creating a 2.5.2 tag on the merge commit of this PR

Reply:
Second thoughts on this: #24